apps/builder/app/routes/_ui.$.tsx

```diff
@@ -32,24 +32,23 @@ export const loader = async ({ request }: LoaderFunctionArgs) => {
 
   if (contentType?.includes("application/json")) {
     // Return an error to not trigger the ErrorBoundary rendering (api request)
-    throw new Response(null, {
+    throw new Response("Not found", {
       status: 404,
       statusText: "Not Found",
```

Comment:
Does it still need statusText? looks duplicated

Reply:
In case of some errors statusText usually is still visible in network panel.
And in some cases it's replaced with default like "Forbidden" - I haven't got the exact bahviour right now, gonna to leave all untill I have some knowledge.

Reply:
Seems like current behavior
- UI routes hide server statusText
- API routes shows mine statusText